_data/providers.json

```
@@ -303,6 +303,11 @@
       "url": "https://nodecraft.com",
       "description": "Use the default server port and `0.0.0.0` or your server IP as the host address."
     },
+    {
+      "name": "PaperNodes",
+      "url": "https://papernodes.com/",
+      "description": "Enable clone-remote-port (or manually set the Bedrock port to the Java port), and connect with the Java IP and port or you can contact us to request an additional port or a dedicated IP."
```

```suggestion
      "description": "Enable clone-remote-port (or manually set the Bedrock port to the Java port), and connect with the Java IP and port. Alternatively, you can contact the host to request an additional port or a dedicated IP."
```
Since these instructions would be shown on our wiki, people might understand "us" as being Geyser, not your host.

---

Done